### ISSUE TITLE
Run tests in different databases for each browser

### ISSUE THREAD

**Issue by warpech** [Starcounter/UniformDocs#387]
With PRs https://github.com/Starcounter/UniformDocs/pull/384 and https://github.com/Starcounter/UniformDocs/pull/386, the tests for UniformDocs will become very slow unless we find a way to paralellize.

I think that the low hanging fruit to paralellize is to start a separate Starcounter database for each browser. Then we can test all browsers in parallel, without tests affecting each other.

**Comment by warpech** [comment]
Recently I think that the solution proposed in OP is rather a workaround than an elegant fix. Closing in favour of #396
